common/artifacts/000_wc_artifacts.txt

```diff
@@ -838,6 +1314,156 @@ prosthetic_blade_hand = {
 
 	picture = "GFX_inv_weapon_shortblade_29"
 }
+warsong_battle_drums = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		
+	}
+
+	stacking = yes
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+	command_modifier = {
+		heavy_infantry = 0.05
+		light_infantry = 0.05
+		archers = 0.05
+		light_cavalry = 0.05
+		morale_offence = 0.1
+	}
+	picture = "GFX_warsong_battle_drums"
+	
+	slot = leader
+}
+scourge_bone_chimes = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes	
+			OR = {
+			is_death_knight_class_trigger = yes
+			is_necromancer_class_trigger = yes
+			is_warlock_class_trigger = yes
+			OR = {
+				religion = death_god
+				religion = fel_group
+			}
+		}
+	}
+
+	stacking = yes
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+
+	general_opinion = -5
+	death_god_opinion = 5
+	command_modifier = {
+		morale_defence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_scourge_bone_chimes"
+	
+	slot = leader
+}
+frost_wyrm_skull_shield = {
```

Comment:
So, you can use shield + two-handed weapon, shield + staff, etc?

Reply:
for now, yes. I didn't make any more shields so far, because I don't know how to solve 2H/2x1H/1H+Shield yet...

Reply:
or could delete personal combat score from shield and make it a decorative one

Reply:
made into relic

Reply:
made into relic